maintainers/maintainer-list.nix

```
@@ -4520,6 +4520,11 @@
     github = "minijackson";
     name = "Rémi Nicole";
   };
+  mir06 = {
+    email = "armin.leuprecht@uni-graz.at";
+    githup = "mir06";
```

```suggestion
    github = "mir06";
```

And please add there your github ID - it's available here - https://caius.github.io/github_id/

---

Oh another typo. Thx!